.github/renovate.json

```diff
@@ -5,6 +5,9 @@
   ],
   "enabledManagers": ["gomod", "cargo"],
   "reviewers": ["codingllama", "jentfoo", "rosstimothy", "zmb3"],
+  "branchConcurrentLimit": 10,
```

Comment:
Should we give it a bit more leeway? Our number of modules is increasing fast and this toggle here is easy to forget.